src/nvme/tree.h

```diff
@@ -452,6 +452,14 @@ const char *nvme_ns_get_sysfs_dir(nvme_ns_t n);
  */
 const char *nvme_ns_get_name(nvme_ns_t n);
 
+/**
+ * nvme_ns_get_chrdev_name() -
+ * @n:
+ *
+ * Return: 
+ */
+const char *nvme_ns_get_chrdev_name(nvme_ns_t n);
```

Comment:
And you need to update libnvme.map to expose the function.

Reply:
Missed that one.  Will do this. Thanks.